sensor_covariance_fusion:
  # yaw is 100 % accurate
  imu_orientation: 
    [
      0,    0,    0,     
      0,    0, 0,    
      0,    0,    0.0, 

    ]

  imu_angular_velocity: 
    [
      1.0, 0,    0,     
      0,    1.0, 0,    
      0,    0,    1.0, 

    ]
  # 0.001, 0.001, 0.001 (From std-dev)

  imu_linear_acceleration: 
    [
      1.0, 0,    0,     
      0,    1.0, 0,    
      0,    0,    1.0, 

    ]
  # 0.001, 0.001, 0.0015 (From std-dev)

  gps_position:
    [
      1.0, 0,    0,     
      0,    1.0, 0,    
      0,    0,   1.0, 

    ]


#https://github.com/carla-simulator/leaderboard/blob/leaderboard-2.0/leaderboard/autoagents/agent_wrapper.py
# attributes['noise_alt_stddev'] = str(0.000005)
# attributes['noise_lat_stddev'] = str(0.000005)
# attributes['noise_lon_stddev'] = str(0.000005)
# attributes['noise_alt_bias'] = str(0.0)
# attributes['noise_lat_bias'] = str(0.0)
# attributes['noise_lon_bias'] = str(0.0)

# attributes['noise_accel_stddev_x'] = str(0.001)
# attributes['noise_accel_stddev_y'] = str(0.001)
# attributes['noise_accel_stddev_z'] = str(0.015)
# attributes['noise_gyro_stddev_x'] = str(0.001)
# attributes['noise_gyro_stddev_y'] = str(0.001)
# attributes['noise_gyro_stddev_z'] = str(0.001)
